coda/coda_mdstore/urls.py

```diff
@@ -11,6 +11,10 @@
         r'^bag/(?P<identifier>ark:\/\d+\/.+?)/links/$', views.bagURLList,
         kwargs={'html': True}, name='bag-links'
     ),
+    re_path(
+        r'^bag/(?P<identifier>ark:\/\d+\/.+?)/download/$', views.bagURLList,
+        kwargs={'download': True}, name='bag-downloads'
```

Comment:
How about singular 'bag-download' for the `name` since it is downloading a single bag?

Reply:
it might make sense to have the path for these downloadable zip files something like `http://coda.library.unt.edu/bag/ark:/67531/coda5kd3.zip` 

I don't think we are 100% consistent in the URL patterns in coda and that is probably beyond the scope here.  For me it would mirror the `.urls` files we return.